backend/src/api/users/users.api.ts

```diff
@@ -22,6 +22,14 @@ const initUsersApi = ({ apiRouter, userService }: Args): Router => {
     }),
   );
 
+  userRouter.get(
+    UsersApiPath.$ID,
+    checkAuth(HttpMethod.GET),
+    handleAsyncApi(async (_req, res) => {
```

Comment:
```suggestion
    handleAsyncApi(async (req, res) => {
```
underscore only for unused params